lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -19,31 +19,6 @@ class Console::CommandDispatcher::Stdapi::Sys
   include Console::CommandDispatcher
   include Rex::Post::Meterpreter::Extensions::Stdapi
 
-  #
-  # Options used by the 'execute' command.
-  #
-  @@execute_opts = Rex::Parser::Arguments.new(
-    "-a" => [ true,  "The arguments to pass to the command."		   ],
-    "-c" => [ false, "Channelized I/O (required for interaction)."		   ], # -i sets -c
-    "-f" => [ true,  "The executable command to run."			   ],
-    "-h" => [ false, "Help menu."						   ],
-    "-H" => [ false, "Create the process hidden from view."			   ],
-    "-i" => [ false, "Interact with the process after creating it."		   ],
-    "-m" => [ false, "Execute from memory."					   ],
-    "-d" => [ true,  "The 'dummy' executable to launch when using -m."	   ],
-    "-t" => [ false, "Execute process with currently impersonated thread token"],
-    "-k" => [ false, "Execute process on the meterpreters current desktop"	   ],
-    "-z" => [ false, "Execute process in a subshell"	   ],
-    "-s" => [ true,  "Execute process in a given session as the session user"  ])
-
-  #
-  # Options used by the 'shell' command.
-  #
-  @@shell_opts = Rex::Parser::Arguments.new(
-    "-h" => [ false, "Help menu."                                          ],
-    "-l" => [ false, "List available shells (/etc/shells)."                ],
-    "-t" => [ true,  "Spawn a PTY shell (/bin/bash if no argument given)." ]) # ssh(1) -t
-
```

Comment:
Just to make the git blame cleaner, we could revert this change, and just add an extra class variable with the extra options merged in:
```
      @@shell_opts_with_fully_interactive_shell = Rex::Parser::Arguments.new(@@shell_opts.fmt.merge(
        { '-i' => [ false, 'Drop into a fully interactive shell. (Only used in conjunction with `-t`'] }
      ))
```

And update the accessor methods to reference this instead. When the feature is added for real, the git diff will just be deleting the accessor methods, and adding the `-i` to the existing `@@shell_opts` which is easier to review.